support/integration-testing/docker-compose.bruno.general.yml

```diff
@@ -0,0 +1,159 @@
+
+networks:
+  main:
+
+services:
+  bruno:
+    image: node:24-bullseye
+    pull_policy: 'missing'
+    volumes:
+      - "../../.tmp/junit/backend-bruno:/opt/junit/backend-bruno"
+      - "../../test/bruno/bruno.json:/opt/bruno-tests/bruno.json"
+      - "../../test/bruno/package.json:/opt/bruno-tests/package.json"
+      - "../../test/bruno/package-lock.json:/opt/bruno-tests/package-lock.json"
+      - "../../test/bruno/attachment.png:/opt/bruno-tests/attachment.png"
+      - "../../test/bruno/environments:/opt/bruno-tests/environments"
+      - "../../test/bruno/1 - REST API:/opt/bruno-tests/1 - REST API"
+      - "../../test/bruno/2 - AJAX calls:/opt/bruno-tests/2 - AJAX calls"
+      - "../../test/bruno/3 - OpenID Connect:/opt/bruno-tests/3 - OpenID Connect"
+      - "../../test/bruno/4 - General:/opt/bruno-tests/4 - General"
+    working_dir: /opt/bruno-tests
+    stop_grace_period: 1800s
+    depends_on:
+      redboxportal:
+        condition: service_healthy
+    networks:
+     main:
+       aliases:
+         - bruno
+    entrypoint: >-
+      /bin/sh -lc "cd /opt/bruno-tests &&
+      npm install &&
+      npx @usebruno/cli@1.34.1 run
+      --env int-test
+      --format junit
+      --output /opt/junit/backend-bruno/backend-bruno-general.xml
+      --bail;
+      status=$?;
+      wget -qO- http://redboxportal:1599 || true;
+      sleep 180;
+      exit $status"
+  redboxportal:
+    image: qcifengineering/redbox-portal:feature-node24
+    pull_policy: 'missing'
+    user: root
+      # Debugging port (publish only when debugging locally)
+      # ports:
+      #   - "9876:9876"
+    volumes:
+       - "../..:/opt/redbox-portal"
+       - "../../.tmp/attachments:/attachments"
+       - "../../test/resources/config/emailnotification.js:/opt/redbox-portal/config/emailnotification.js"
+    stop_grace_period: 1800s
```

Comment:
Change `stop_grace_period: 1800s` to `stop_grace_period: 30m` to make it clearer.